arch/arm/core/aarch64/Kconfig

```diff
@@ -56,6 +56,9 @@ config CMSIS_V2_THREAD_DYNAMIC_STACK_SIZE
 config IPM_CONSOLE_STACK_SIZE
 	default 2048
 
+config MMU_PAGE_SIZE
```

Comment:
Maybe add "if ARM_MMU", I think it's linked

Reply:
Is that needed? `MMU_PAGE_SIZE` is defined only if `MMU` is enabled and `MMU` is selected by `ARM_MMU`.

Anyway, I'm moving that into the `if ARM_MMU` section in the same for the sake of consistency.